app-games/src/main/java/cm/aptoide/pt/app_games/notifications/NotificationUtils.kt

```diff
@@ -0,0 +1,5 @@
+package cm.aptoide.pt.app_games.notifications
+
+import cm.aptoide.pt.app_games.R
+
+fun String.getNotificationIcon() = R.drawable.ic_notification_icon
```

Comment:
This notification icon for now is always the same and is not dependent on the build type. This function can be removed